--- a/corec/corec/CMakeLists.txt
+++ b/corec/corec/CMakeLists.txt
@@ -52,7 +52,6 @@
 else()
 	list(APPEND COREC_SOURCES
 		helpers/date/date_libc.c
-		helpers/file/file_libc.c
 		helpers/file/file_bctbx.c
 		multithread/multithread_pthread.c
 	)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,6 +53,7 @@
 )
 
 set(LINK_FLAGS )
+link_libraries(bctoolbox)
 if(APPLE)
 	list(APPEND LINK_FLAGS "-framework Foundation")
 	if(IOS)
